tools/Environments/DevHome.Environments/Strings/en-us/Resources.resw

```diff
@@ -190,4 +194,28 @@
     <value>All</value>
     <comment>Text for the default value of all providers for filtering</comment>
   </data>
+  <data name="RemoveButtonTextForCreateComputeSystem" xml:space="preserve">
+    <value>Remove</value>
+    <comment>Text for the remove button when the user wants to remove an environment being created from the UI</comment>
+  </data>
+  <data name="FailureMessageForCreateComputeSystem" xml:space="preserve">
+    <value>Failed to create environment due to error: {0}</value>
+    <comment>Locked="{0}" Text to show the the user when Dev Home receives the notification that the creation of the new environment has failed. {0} is the error that is displayed to the user</comment>
+  </data>
+  <data name="SuccessMessageForCreateComputeSystem" xml:space="preserve">
+    <value>The {0} provider has completed its creation steps. Re-sync the page to manage you're new environment</value>
+    <comment>Locked="{0}" Text to show the the user when Dev Home receives the notification that the users new environment has been created.  {0} is the of the provider who sent the notification</comment>

```

Comment:
What does "Locked" mean here (and in other places where it's used in a sentence with parameters)?

Reply:
It was to tell localizers not to localize whats in {0}. We've been using it in Dev Home but I do see [this in osg](https://www.osgwiki.com/wiki/Source_Commenting_for_Apps_and_Store_projects#Do_I_need_to_protect_managed_and_native_placeholders?) which looks like we don't need to do it. So far, the localizers haven't said anything but I guess we can probably stop. I'll follow up with Eric/Kristen to confirm. Then maybe we can remove them from the codebase. for now i'll keep it